kitchen.recipes.yml

```diff
@@ -9,23 +9,24 @@ verifier:
   name: inspec
   inspec_tests:
     # recipe tests will use controls from these directories
-    - test/recipes
+    - test/recipes/aws-parallelcluster-install
```

Comment:
This is redundant: the controls are looked up in all the subfolders

Reply:
cannot make it work like that

Reply:
We will sync on that offline